package.json

```diff
@@ -47,15 +47,16 @@
     "xtend": "4.0.0"
   },
   "devDependencies": {
+    "@uber/eslint-config": "^3.0.1",
```

Comment:
I dont think this is open source ...

You should just use https://github.com/raynos/eslint-config-perf-standard instead.